__tests__/utils.test.js

```diff
@@ -0,0 +1,36 @@
+import {validation as v} from '../src/js/utils';
+
+const testValues = [null, 5, undefined, {}, '', NaN];
+describe('Testing validation', () => {
+
+  test('isNull works correcty', () => {
```

Comment:
same here: _isNull works correctly_ does not add meaning.
better would be _isNull returns true for `null`_

and the test would be better written as
`expect(isNull(null)).toBe(true)`
`expect(isNull(undefined)).toBe(false)` 

I know that you're trying to test all the values at once. but it makes the test harder to read. 
you have 2 expectations and 5 values you would be testing for, right?

Reply:
not following, should I then test all 5 values separately? I would have to do the same for each function which would make the file much longer and harder to read/change/etc

Reply:
there is something similar to parameterized tests for python for jest as well:

```js
  test.each([
    [null, true],
    [undefined, false],
    [{}, false],
    ['', false],
    [NaN, false],
  ])('.isNull(%s)', (value, expected) => {
    expect(v.isNull(value)).toBe(expected);
  });
```

Reply:
that's actually quite nice - I'll make the change